packages/slice-machine/lib/models/common/widgets/RepeatableGroup/Form.jsx

```diff
@@ -0,0 +1,29 @@
+import { Col, Flex as FlexGrid } from "@components/Flex";
```

Comment:
✍️ potential change

Can we not reuse Form component from the non-repeatable Group?

Reply:
We can, I just forgot! It's done now.